README.md

```diff
@@ -1,65 +1,49 @@
-# Base template for Gulp
+# Tree from object
 
-Template to start from before starting frontend projects.
+## Task
 
-## Install template
+Write a function `createTree` that creating list of `ul/li` from object.
 
-- Create empty repository for your project. [learn how](#) _need a link here_
-- Clone boilerplate into project directory and navigate there.
-```bash
-$ git clone https://github.com/mate-academy/gulp-template.git project-name
-$ cd project-name
-```
-- Change remote repository to recently created
-```bash
-$ git remote set-url origin github-link-of-project
-```
-- Install dependencies
-```bash
-$ npm install
-```
+Function `createTree(element, data)`
 
-All packages will be installed after executing the script above.
+`element` - is a DOM element.
 
-## Development mode 
-
-```bash
-$ npm run start
-```
-
-## Build the project
+## Workflow
```

Comment:
We should extract this repetitive workflow into one guidelines file.

Reply:
Created new repo - https://github.com/mate-academy/js_task-DOM-guideline
removed workflow and add a link to guideline